podvm/Makefile.inc

```diff
@@ -121,6 +123,6 @@ $(ATTESTATION_AGENT_SRC):
 	git clone "$(ATTESTATION_AGENT_REPO)" "$(ATTESTATION_AGENT_SRC)"
 
 $(ATTESTATION_AGENT): $(ATTESTATION_AGENT_SRC)
-	cd "$(ATTESTATION_AGENT_SRC)" && $(RUST_FLAGS) $(MAKE) KBC="$(AA_KBC)" LIBC="$(LIBC)"
+	cd "$(ATTESTATION_AGENT_SRC)" && CC= ARCH=$(ARCH) $(MAKE) KBC="$(AA_KBC)" LIBC="$(LIBC)"
```

Comment:
is this `CC=` correct ?

Reply:
Yes, attestation agent includes `libgit2` library which doesn't build when specifying `CC`.